views/intranet.ejs

```diff
@@ -126,6 +127,18 @@ this license in a file with the distribution.
             <button class="button">View jobs</button>
           </a>
         </div>
+      <div id="main-center" class="small-12 medium-12 large-4 columns">
+          <h3>Credits</h3>
+          <p>View and refill your ACM credits balance</p>
+          <a href="/credits">
+            <button class="button">View credits</button>
+          </a>
+          <% if(session.roles.isAdmin) { %>
```

Comment:
Allow top 4 / corporate to access? @tyler-thetyrant probably wants access to this